pycobertura/reporters.py

```diff
@@ -58,6 +58,11 @@ def generate(self):
         lines = self.get_report_lines()
         return tabulate(lines, headers=headers_with_missing)
 
+class MarkdownReporter(Reporter):
```

Comment:
We'll need extra space to get the tests to pass, I think.

```suggestion

class MarkdownReporter(Reporter):
```